### PR TITLE
Маячкин - Лабораторная работа #1

### PR DESCRIPTION
**Pull request by senyamac**

### CODE REVIEW

**Comment by senyamac** [comment]
@l-bat
@GerasimovDG
@MaksimenkoAN

File: code/stack/model/stack.py [hunk 1]
```diff
@@ -0,0 +1,42 @@
+class Stack:
+    max_size = 100
```
Comment (l-bat) [inline code/stack/model/stack.py:2]: It looks like `max_size` and `self. max_size` are different variables. I think `self. max_size`  in constructor is enough.

Comment (senyamac) [reply code/stack/model/stack.py]: Thanks, I have applied your tip.

**Comment by l-bat** [comment]
Thanks :+1:!

**Review by l-bat — APPROVED** [review]

File: code/stack/model/stack.py [hunk 2]
```diff
@@ -0,0 +1,41 @@
+class Stack:
+    def __init__(self, size=None):
+        self.stack = []
+        self.max_size = 100
+        if size is not None:
+            if type(size) is not int:
```
Comment (GerasimovDG) [inline code/stack/model/stack.py:6]: maybe `if not isinstance(size, int):` will be better?

Comment (senyamac) [reply code/stack/model/stack.py]: All done. I think it's looking good, thanks.

**Review by GerasimovDG — APPROVED** [review]

File: code/stack/model/stack.py (hunk 2 above)
Comment (GerasimovDG) [reply code/stack/model/stack.py]: Good. 👍

**Comment by MaksimenkoAN** [comment]
👍

**Comment by senyamac** [comment]
@kirill-korniakov

**Pull request merged**